src/index.ts

```diff
@@ -183,11 +191,14 @@ export type GetAddressInfoResult = {
 }
 
 export type ImportMultiRequest = {
-  scriptPubKey: string | { address: string }
+  desc?: string
+  scriptPubKey?: string | { address: string }
   timestamp: number | 'now'
   redeemScript?: string
+  witnessScript?: string
   pubkeys?: string[]
   keys?: string[]
+  range: [number, number]
```

Comment:
based on the docs (https://bitcoincore.org/en/doc/0.20.0/rpc/wallet/importmulti/) shouldn't that be `number | [number, number]`?